### PR TITLE
Package Creation for Okta Single signOn

### PR DESCRIPTION
**Pull request by v-spadarthi**
Change(s):

See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution Okta Single signOn

See guidance below
Version Updated:
2.0.0

See guidance below
Testing Completed:
The link for test results is 
https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7BAD683178-1D03-4E0C-BD17-34295EEF2BF5%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20Okta%20Single%20Sign%20On.docx&action=default&mobileredirect=true

See guidance below
This is an 3P solution.

### CODE REVIEW

**Comment by v-dvedak** [comment]
@v-spadarthi, please ensure 1.0.1 version zip is deleted but it should not be so.

**Comment by v-marimanda** [comment]
Hi @v-sabiraj
Can you please get confirmation on this PR merging or closing and act accordingly.
Thanks

**Comment by NikTripathi** [comment]
@v-sabiraj Please look into this.

**Comment by v-mchatla** [comment]
Hi @v-spadarthi 
Please address the comments provided by Darshana by reverting the changes for zip file 1.0.1
Thanks

**Comment by v-marimanda** [comment]
Hi @v-spadarthi
Please address the comments provided by Darshana by reverting the changes for zip file 1.0.1.
Thanks

**Comment by v-marimanda** [comment]
Hi @v-sabiraj and @v-atulyadav 
Can you please take look once and provide your sign off. thanks.
Thanks

**Comment by v-marimanda** [comment]
Hi @v-sabiraj and @v-atulyadav
Can you please take look once and provide your sign off. thanks.
Thanks

**Comment by v-marimanda** [comment]
@v-atulyadav Is their any update for this pr. Please mention in the comment. thanks!!!

**Comment by v-marimanda** [comment]
@v-atulyadav Is their any update for this pr. Please mention in the comment. thanks!!!

**Comment by v-amolpatil** [comment]
@v-atulyadav  Is their any update for this pr. Please mention in the comment. thanks!!!

**Comment by v-marimanda** [comment]
@v-atulyadav  Can you please update status for pr?

**Comment by v-marimanda** [comment]
@v-spadarthi Please add the skip validations.

File: Solutions/Okta Single Sign-On/SolutionMetadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"publisherId": "azuresentinel",
 	"offerId": "azure-sentinel-solution-okta",
 	"firstPublishDate": "2022-03-24",
-	"providers": ["Microsoft "],
+	"providers": ["Okta "],
```
Comment (v-atulyadav) [inline Solutions/Okta Single Sign-On/SolutionMetadata.json:5]: Hi @v-spadarthi, please remove space from provider.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**